packages/voting-contract/src/state.rs

```diff
@@ -47,6 +47,58 @@ impl<P> From<Proposal<P>> for ProposalInfo {
     }
 }
 
+impl<P> Proposal<P> {
```

Comment:
I assume this was just moved and is unmodified.

Reply:
I explained that in description, it's just moved.